dashboard/Makefile

```diff
@@ -2,5 +2,5 @@
 src/shared/hapi/release.js:
 	./node_modules/.bin/pbjs -t static-module -w commonjs -o src/shared/hapi/release.js -p $$GOPATH/src/k8s.io/helm/_proto $$GOPATH/src/k8s.io/helm/_proto/hapi/release/release.proto && \
 	./node_modules/.bin/pbts -o src/shared/hapi/release.d.ts src/shared/hapi/release.js && \
-	echo "// tslint:disable\n$$(cat src/shared/hapi/release.js)" > src/shared/hapi/release.js && \
-	echo "// tslint:disable\n$$(cat src/shared/hapi/release.d.ts)" > src/shared/hapi/release.d.ts
+	echo "/* eslint-disable */\n$$(cat src/shared/hapi/release.js)" > src/shared/hapi/release.js && \
+	echo "/* eslint-disable */\n$$(cat src/shared/hapi/release.d.ts)" > src/shared/hapi/release.d.ts
```

Comment:
This is a make task to generate the file `hapi/release` (that basically defines the spec of a chart). It has not been modified since its creation :shrug: